OutOfSchool/OutOfSchool.BusinessLogic/Services/CurrentUserService.cs

```diff
@@ -228,7 +228,7 @@ private async Task<bool> ProviderHasRights(Guid providerId)
 
     private async Task<bool> ProviderAdminHasRights(string providerAdminId)
     {
-        if (!this.IsEmployeeOrProvider() || UserId != providerAdminId)
+        if (!this.IsEmployee() || UserId != providerAdminId)
```

Comment:
@VadymLevkovskyi We are transitioning to a more generic approach and instead of `ProviderAdmin` we are going to have `Employee` and `Position` to check what an employee can do. For now this is the first step - renaming enities and removing excessive logic tied to `ProviderAdmin` & `ProviderDeputy`. This method right here will be changed soon and all the naming will be streamlined.